nixos/modules/config/users-groups.nix

```diff
@@ -960,12 +979,16 @@ in {
           (filter (x: x != null) (map (flip getAttr user) passwordOptions));
       in optional (!unambiguousPasswordConfiguration) ''
         The user '${user.name}' has multiple of the options
-        `hashedPassword`, `password`, `hashedPasswordFile`, `initialPassword`
-        & `initialHashedPassword` set to a non-null value.
+        `initialHashedPassword`, `hashedPassword`, `initialPassword`, `password`
+        & `hashedPasswordFile` set to a non-null value.
         The options silently discard others by the order of precedence
```

Comment:
I guess this is not true because of the mutableUsers true/false difference, so the order of precedence isn't necessary as shown above. Would be nice to refer to a non-duplicated source of truth here as well, so maybe I'll add a separate variable with the order for mutable false/true text and then include that both in passwordDescription and here.. 

I also wonder if the order of precedence difference between mutableUsers true/false is intentional or accidental. If the former, I wonder who would know, and if the latter, I think it should just be changed for a future release so the orders are consistent?